Dockerfile

```diff
@@ -12,20 +12,24 @@ RUN apt install bluez -y
 
 RUN apt install build-essential libglib2.0-dev libical-dev libreadline-dev libudev-dev libdbus-1-dev libdbus-glib-1-dev bluetooth libbluetooth-dev usbutils -y
 
-RUN pip3 install idasen-controller==2.0.1
+RUN export SKIP_CYTHON=false
 
-RUN apt install curl -y
+RUN pip3 install idasen-controller==2.2.0
+
+RUN apt-get update
```

Comment:
Why the change to apt-get?

Reply:
Old habits, sorry! apt is fine but needs the `apt update` command to fix the error 404 when apt looks for packages list.

Reply:
Have to check again, but I believe that's old habits.